event_routing_backends/processors/xapi/constants.py

```diff
@@ -0,0 +1,72 @@
+"""
+Constants for xAPI specifications.
+"""
+
+# xAPI verbs
+XAPI_VERB_ATTEMPTED = 'http://adlnet.gov/expapi/verbs/attempted'
+XAPI_VERB_ANSWERED = 'http://adlnet.gov/expapi/verbs/answered'
+# XAPI_VERB_FAILED = 'http://adlnet.gov/expapi/verbs/failed'
```

Comment:
Why are these commented out?

Reply:
removed these as they were not used anywhere